### PR TITLE
[ACCT-2125] test(acbc): adding us AC BC postman collection

### PR DESCRIPTION
**Pull request by TruptiShrimant**
Adding/modifying AC and BC postman collection.

### CODE REVIEW

**Review by AnDyXX — APPROVED** [review]

File: accounts/BalanceCheck.postman_collection.json [hunk 1]
```diff
@@ -699,5 +676,61 @@
...
+			"key": "client_secret",
+			"value": "1f9b258af2ec40159eda8c9bda750e27"
+		},
+		{
+			"key": "report_id",
+			"value": "{{report_id}}"
+		},
+		{
+			"key": "access_token",
+			"value": "eyJhbGciOiJFUzI1NiIsImtpZCI6ImRiODdiNzViLTY2NTUtNGYzYy04MTliLTNkMjU0NTliMjVmZiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3MDY1NDMyOTcsImlhdCI6MTcwNjUzNjA5NywiaXNzIjoidGluazovL2F1dGgiLCJqdGkiOiJjMGIxYzQyYi0yYjZlLTQ0YWMtYWZmMy0zMDE5ZWNjN2RjY2EiLCJvcmlnaW4iOiJtYWluIiwic2NvcGVzIjpbImNyZWRlbnRpYWxzOnJlZnJlc2giLCJhY2NvdW50cy5iYWxhbmNlczpyZWFkb25seSIsImFjY291bnRzOnJlYWQiLCJiYWxhbmNlLXJlZnJlc2giLCJjcmVkZW50aWFsczpyZWFkIl0sInN1YiI6InRpbms6Ly9hdXRoL3VzZXIvMWY3ZDhkN2NjMDM4NDBmMjg1MTVjNzFiNzkzNTNkMjEiLCJ0aW5rOi8vYXBwL2lkIjoiYWY4MWMzMzAwZmYzNDAxNGIyMmNhNjhlZDMyZGMzMDIiLCJ0aW5rOi8vYXBwL3ZlcmlmaWVkIjoidHJ1ZSIsInRpbms6Ly9jbGllbnQvaWQiOiJhZTg5YzYzOGQ1MjQ0NGJmYmVhMDUwYTZjZDc3M2QyOSJ9.ruytDwqX6saMERfJbwAuT02KEcYJDjIXKu4DcEQ7AlEhB1xD5zp8Ll9CXVYdRYy3NZixWf-Bt2acrS0yE_zBDw"
```
Comment (tinkjorgen) [inline accounts/BalanceCheck.postman_collection.json:705]: is this supposed to be hard coded auth token?

File: accounts/BalanceCheck.postman_collection.json [hunk 2]
```diff
@@ -699,5 +676,61 @@
...
+			"key": "market",
+			"value": "UK"
+		},
+		{
+			"key": "code",
+			"value": "da1969ad2f364205a870faa62136b92e"
+		},
+		{
+			"key": "tink_link",
+			"value": "https://link.tink.com/1.0/account-check/connect?client_id=ae89c638d52444bfbea050a6cd773d29&state=OPTIONAL&redirect_uri=https%3A%2F%2Fconsole.us.tink.com%2Fcallback&authorization_code=415ae98740df403b9d202984006f8eba&market=US"
```
Comment (tinkjorgen) [inline accounts/BalanceCheck.postman_collection.json:725]: hardcoded client id and code?

File: accounts/Production.postman_environment.json [hunk 3]
```diff
@@ -1,109 +1,129 @@
...
+			"type": "any",
 			"enabled": true
 		}
 	],
 	"_postman_variable_scope": "environment",
-	"_postman_exported_at": "2024-03-12T15:42:33.494Z",
-	"_postman_exported_using": "Postman/7.36.7"
-}
+	"_postman_exported_at": "2024-04-19T13:28:11.658Z",
+	"_postman_exported_using": "Postman/10.22.13"
```
Comment (tinkjorgen) [inline accounts/Production.postman_environment.json:128]: should we upgrade to postman 10/11?

File: accounts/BalanceCheck.postman_collection.json [hunk 4]
```diff
@@ -699,5 +676,61 @@
...
+			"disabled": true
+		},
+		{
+			"key": "external_user_id",
+			"value": "external_user_id",
+			"disabled": true
+		},
+		{
+			"key": "client_id",
+			"value": "040bf30d13bf41499d1b92bc533712d2"
```
Comment (tinkjorgen) [inline accounts/BalanceCheck.postman_collection.json:693]: hardcoded?

File: accounts/BalanceCheck.postman_collection.json [hunk 5]
```diff
@@ -699,5 +676,61 @@
...
+		},
+		{
+			"key": "client_secret",
+			"value": "1f9b258af2ec40159eda8c9bda750e27"
```
Comment (tinkjorgen) [inline accounts/BalanceCheck.postman_collection.json:697]: hardcodede

**Pull request merged**